src/objects/zcl_abapgit_object_apis.clas.abap

```diff
@@ -0,0 +1,184 @@
+CLASS zcl_abapgit_object_apis DEFINITION
+  PUBLIC
+  INHERITING FROM zcl_abapgit_objects_super
+  FINAL
+  CREATE PUBLIC.
+
+  PUBLIC SECTION.
+
+    INTERFACES zif_abapgit_object.
+
+    METHODS:
+      constructor
+        IMPORTING
+          is_item     TYPE zif_abapgit_definitions=>ty_item
+          iv_language TYPE spras.
+
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+    DATA mo_handler TYPE REF TO object.
+
+    METHODS initialize.
+
+ENDCLASS.
+
+
+
+CLASS ZCL_ABAPGIT_OBJECT_APIS IMPLEMENTATION.
+
+
+  METHOD constructor.
```

Comment:
https://github.com/abaplint/abaplint/issues/1417 😉

Reply:
yea, work in progress, but will add the rule sometime